week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -9,26 +9,40 @@
 
   Use the tests to confirm which message to return
 */
+function funcMap(index) {
+  if (index !== 'pink') {
+    return 'Toxic! Leave bush alone!';
+  } else {
+    return 'Bush is safe to eat from';
+  }
+}
 
-function bushChecker() {
-
+function bushChecker(paramArr) {
+  console.log(paramArr.filter(funcMap));
+  return paramArr.filter(funcMap);
```

Comment:
The objective here is to determine whether the bush has *any* pink berries. 

Hint: Can you *find* any pink berries in your bush?